### PR TITLE
Jt/quickstart installing js

### PR DESCRIPTION
**Pull request by JT1989**
Finished publishing the article, **Quickstart guide: Install Kissmetrics’ JavaScript tracking code, create custom Events and Properties, and identify users**

### CODE REVIEW

File: _posts/2015-10-19-quickstart-install-kissmetrics.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+In this post, we'll be go through each step you need to take to: (1) get the JavaScript code on every page of your site, (2) understand what the Kissmetrics JavaScript snippet tracks, (3)  create custom events and properties, and (4) identify users of your product or service.
+
+* Table of Contents
+{:toc}
+***
+
+##Four Ways to Edit Your Website’s Source Code
+There are plenty of ways to edit your website's source code:
+###Method 1 - Use Git
```
Comment (eskfung) [inline _posts/2015-10-19-quickstart-install-kissmetrics.md:18]: Needs a blank new line between the last sentence and `###Method 1 ...`. Right now it's all being interpreted as one paragraph:

screen shot 2015-10-20 at 12 13 12

File: _posts/2015-10-19-quickstart-install-kissmetrics.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+You can make source code changes through git and push it to your web server. 
+###Method 2 - Download all your HTML files from the web server, edit them in a text editor, and re-upload them to the web server.
```
Comment (eskfung) [inline _posts/2015-10-19-quickstart-install-kissmetrics.md:21]: Same as above.

File: _posts/2015-10-19-quickstart-install-kissmetrics.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+You can download the HTML files via FTP from your web server. Once downloaded, make changes to the code for every page, and re-upload them back to the web server. 
+###Method 3 - Using Content Management Systems (CMS)
```
Comment (eskfung) [inline _posts/2015-10-19-quickstart-install-kissmetrics.md:24]: Same as above.

File: _posts/2015-10-19-quickstart-install-kissmetrics.md (hunk 1 above)
Comment (eskfung) [inline _posts/2015-10-19-quickstart-install-kissmetrics.md:18]: http://daringfireball.net/projects/markdown/syntax#p

File: _posts/2015-10-19-quickstart-install-kissmetrics.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+Choose whichever method works best for you, and then grab your tracking script. 
+##Where to Find Your Kissmetrics JavaScript Snippet
+Your unique tracking code can be found in the settings of your Kissmetrics account. 
+
+###Step 1 - Go To Product Settings
+Go to any page within the Kissmetrics web app and click on Settings:
+
+![Install Kissmetrics][1]
+###Step 2 - Copy the JavaScript Tracking Code to Your Clipboard
```
Comment (eskfung) [inline _posts/2015-10-19-quickstart-install-kissmetrics.md:48]: Same as above.

File: _posts/2015-10-19-quickstart-install-kissmetrics.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+###Method 4 - Make Source Code Changes Through Your Web Host's Console
+If your site is hosted through a traditional managed web hosting service, you may be more comfortable inserting the tracking code through the web console. 
+
+Whichever method you choose, you'll need to make sure you push all code changes live at once. If some pages have the tracking code installed and others don't, you'll get lower quality data. 
+
+Some sites also employ a staging server, which serves as a tool to test the changes made to the site. It's all private to the company, the public doesn't use the staging server. If the changes on the staging server look good, they are then pushed to the production server, where it is live for the public. 
+
+Choose whichever method works best for you, and then grab your tracking script. 
+##Where to Find Your Kissmetrics JavaScript Snippet
```
Comment (eskfung) [inline _posts/2015-10-19-quickstart-install-kissmetrics.md:40]: Same as above.

File: _posts/2015-10-19-quickstart-install-kissmetrics.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+
+###Step 3 - Insert the Tracking Code
+We've copied our JavaScript tracking code and now we need to edit our website's source code. To start, we'll first locate the <head> tag at the top:
+
+![Install Kissmetrics][3]
+Make some space for it and paste the tracking code:
+
+![Install Kissmetrics][4]
+That's all there is to it! After it is live on your server, Kissmetrics will automatically start sending you some events. 
+##What the Kissmetrics JavaScript Snippet Tracks Automatically
```
Comment (eskfung) [inline _posts/2015-10-19-quickstart-install-kissmetrics.md:76]: Same as above.

File: _posts/2015-10-19-quickstart-install-kissmetrics.md [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+
+##Verify Events and Properties Are Working with Kissmetrics Live Feature
+The Kissmetrics Live feature displays in real time the people on your site and the events and properties they are triggering. You can navigate to Live by selecting it in the top navigation.
+
+![Install Kissmetrics][8]
+
+This makes it a great debugger. You'll see if the events and properties you've set up are actually working. And you don't have to rely on other people to trigger events and properties - you can do it yourself. Just go to your site, trigger some events and properties and see if everything in Live is firing correctly.
+
+While Live shows real time data, it'll take an hour for this data to be able to be used in reports.  
+##Identifying People
```
Comment (eskfung) [inline _posts/2015-10-19-quickstart-install-kissmetrics.md:144]: Same as above.

File: _posts/2015-10-19-quickstart-install-kissmetrics.md [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+
+![Install Kissmetrics][9]
+All previous activity while they were under an anonymous ID will merge with their current named ID when they are identified. All the touchpoints Kissmetrics has captured from them before they were identified will be merged into their named ID. You'll see all their pre-purchase activities including what steps led them to converting to registration and paying you.
+
+To learn more about setting up identities, check out our [support document](http://support.kissmetrics.com/getting-started/understanding-identities.html) which covers setting this up. You may need to talk to your developer to get help pulling the identifier information from your back end. 
+
+You'll want to put the identify code anywhere on your website where you identify users. This can be a sign up page, registration page, form field, etc.
+##identifying people: 3 Steps for Installing the Kissmetrics JavaScript
```
Comment (eskfung) [inline _posts/2015-10-19-quickstart-install-kissmetrics.md]: Same as above, and capitalize Identifying.

**Comment by JT1989** [comment]
@eskfung Went through entire article to add new break lines and capitalized certain words. Let me know if I missed anything.

File: _posts/2015-10-19-quickstart-install-kissmetrics.md [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+
+*The "x13" box next to the Returning property indicates the person has returned to our site 13 times since their first visit. When we hover over the box we see when Kissmetrics first saw them return as well as when they most recently returned.* 
+
+You can segment people by their Returning property in a Funnel Report. Here's how that looks:
+
+![Install Kissmetrics][6]
+
+Here we see a typical signup funnel; we're tracking people who visit our site, view the features page and signup. People are segmented by the Returning property. We're seeing how new vs returning visitors convert through the funnel. While most of the people that come to our site are new visitors, we actually get more signups from returning visitors. This tells us that most people make more than one visit before they decide to sign up. 
+
+**Event: Ad Campaign Hit**
```
Comment (eskfung) [inline _posts/2015-10-19-quickstart-install-kissmetrics.md]: Seems like you want a `<br>` here to match the two events below.

So: `**Event: Ad Campaign Hit**<br>`

**Comment by eskfung** [comment]
@JT1989 just one more thing to address and then you're good to go whenever ready. :shipit:

**Comment by JT1989** [comment]
@eskfung good catch! added in the line break. Appreciate your help

**Pull request merged**